client/src/styles/Checkbox.module.css

```diff
@@ -0,0 +1,32 @@
+.checkboxContainer {
+    align-items: center;
+    display: inline-flex;
+    font-family: var(--theme-base-font-family);
+    font-size: var(--theme-base-font-size);
+    user-select: none;
+}
+
+.checkbox {
```

Comment:
I think we should change the active colour of the checkbox, prehaps to the light, bright blue?